docs/user-guide/cli-usingcli.md

```diff
@@ -31,41 +32,331 @@ zowe --ac
+
+### How token management works
+
+When you log in with the CLI, an API ML token is supplied and stored on your computer in place of username and password. The token initiates a secure session with API ML that lets you issue commands for the duration of the session.
+
+**Note:** Zowe CLI also supports standard token implementations such as Java Web Tokens (JWT) and Lightweight Third-Party Authentication (LTPA).
+
+### Logging in
+
+To request a token and initiate a session with API ML, issue the following command:
+
+```
+zowe auth login apiml
+```
+
+The CLI prompts you to enter your username, password (where password can be a PIN concatenated with a second factor for MFA), host, and port for the API ML instance.
+
+**Tip:** If you already created a base profile, you might not be prompted for host and port.
+
+A local base profile is created that contains your token. A secure session is initiated. When you issue commands, you can omit your username, password, host, and port. Instead, you provide a base path and base profile to connect to API ML.
+
+Optionally, if you do not want to store the token on disk, append the `--show-token` option to the login command. If you choose this option, you must manually supply the token on each command using the `--token-value` option.
+
+**Notes:**
+- Tokens expire after a period of time defined by your security administrator. When a token expires, you must log in again to initiate a new session.
+- If you omit connection details from a service profile, such as `zosmf` profile, the CLI uses the information from your base profile.
+- You can choose to specify all connection details on a service profile and connect directly to the service. Routing through API ML is not required.
+
+### Logging out
+
+Log out to end the API ML session and remove the token from your base profile.
+
+Issue the following command:
+
+```zowe auth logout```
+
+### Accessing a service through API ML
+
+To use an active API ML session, specify the following options in your commands and service profiles:
+
+- `--base-path`: The base path of the API ML instance that you want to access.
+- `--base-profile`: The base profile to use, which contains a session token. If you do not specify a base profile, the default base profile is used.
+
+**Note:** To access API ML, ensure that you *do not* provide username, password, host, or port directly on the service commands or profiles. Supplying those options causes the CLI to ignore the token in your base profile and directly access the service.
+
+#### Specifying base path
+
+<!--
+TODO - I'm a little iffy on this section. Are you specifying a base path to zosmf, or just the base path to APIML and it figures out which zosmf instance from there? Generally, is this section correct?
```

Comment:
Looks generally correct to me. Only comment below is regarding the value of the base path.